model/metrics/process-metrics.yaml

```diff
@@ -100,3 +101,19 @@ groups:
     unit: "{fault}"
     attributes:
       - ref: process.paging.fault_type
+
+  - id: metric.process.uptime
+    type: metric
+    metric_name: process.uptime
+    stability: experimental
+    brief: "The number of seconds the process has been running."
+    instrument: counter
+    unit: "s"
+
+  - id: metric.process.args_count
+    type: metric
+    metric_name: process.args_count
```

Comment:
what would be the use case for this metric? Does it make any sense, especially without any attributes?

I.e. I can see P95 of my processes have 2 arguments and P99 is 10 arguments - is this helpful at all?

Reply:
I've taken args_count out for now.  This is useful for threat hunting, but it's really not useful for system performance or other use cases.

In the future we can consider adding it back in a more use-case specific metrics namespace.